build/ci/release.yml

```diff
@@ -151,9 +151,7 @@ functions:
           export GOROOT="${go_root}"
           export PATH="./bin:${go_bin}:$PATH"
 
-          VERSION=$(git describe | cut -d "v" -f 2)
-
-          go run ../internal/release/main.go "${VERSION}" "${FEED_FILE_NAME}"
+          go run ../internal/release/main.go --version "$(git describe | cut -d "v" -f 2)" --file "${FEED_FILE_NAME}"
```

Comment:
git describe may not work anymore with a mono repo

Reply:
Looking at the JSON files generated by the package tasks it seems to work properly

Reply:
I have also made some tests locally and on evergreen and I haven’t found any issue with git describe

Reply:
sorry what I mean is that git describe will get you the latest tag, for now it works since we only release one thing, this may become flaky later

Reply:
Thanks for the clarification!
I don't think this is an issue: you are right that the tag returned by git describe will be different when releasing atlascli v0.0.1 as the `mongocli.json` file will contain the version 0.0.1 when running the packaging tasks. However, this won't happen when running releasing tasks as the latest tag will always be the correct one. Let me know if I am missing something here. Thanks!

Reply:
not missing much and why I say may become flaky, race conditions may happen if releasing both the same day 

Also this leaves us in a position where master json files will always point to latest, ie, when we do 0.0.1 of atlas we'll have the mongocli json with 0.0.1 as well

Reply:
I am going to create a ticket to see how to fix this issue. I will start working on it once I have pre-released atlascli

Reply:
https://jira.mongodb.org/browse/CLOUDP-114080